.travis.yml

```diff
@@ -1,15 +1,29 @@
 language: python
 cache:
   - pip
+addons:
+  hosts:
+    - jupyter.test
```

Comment:
I learned recently from Min that we could use local.jovyan.org instead to point to localhost, and by doing so avoid the need to setup things in /etc/hosts or like we do here for travis.

Suggestion: replace jupyter.test with local.jovyan.org

Reply:
That's really cool :sparkles: Thanks @consideRatio :heart: